### DISCUSSION TITLE
DataGridTemplateColumn copy to clipboard MemberPath?

### DISCUSSION THREAD

**Discussion by sn4k3** [AvaloniaUI/Avalonia#17811] [Q&A] [answered]
I have an DataGrid with `ClipboardCopyMode="IncludeHeader"`
That works with normal column only, but when using a `DataGridTemplateColumn` the copy returns an empty string `""`. For sort there are the `SortMemberPath` but for Copy I can't find a way. The `ClipboardContentBinding` binds to vm rather than row item. Is there a way without code behind?

**Comment by timunie** [comment, marked as answer]
ClipboardContentBinding binds on your row, but compiler may not understand it. When in doubt, just cast to RowViewModel

**Reply by sn4k3** [reply]
Thank you,

I was unable to cast it but works fine when set DataType:

`ClipboardContentBinding="{Binding Order, DataType=network:PingableHost}"`
